pyproject.toml

```
@@ -28,18 +28,20 @@ elasticsearch = "7.13.4"
 elasticsearch-dsl = "^7.0.0"
 gitpython = "^3.1.2"
 pytz = "^2020.1"
```

Using "^" here makes this hard to lock against other repos because `pytz` files after 2020 will be disallowed as incompatible major version bumps.
```suggestion
pytz = ">=2020.1"
```